user_tasks/migrations/0002_default_file_storage.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.2 on 2016-12-05 16:12
+from __future__ import unicode_literals
+
+import django.core.files.storage
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user_tasks', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='usertaskartifact',
+            name='file',
+            field=models.FileField(blank=True, null=True, storage=django.core.files.storage.FileSystemStorage(), upload_to='user_tasks/%Y/%m/%d/'),
```

Comment:
It already does; see https://github.com/edx/django-user-tasks/blob/master/user_tasks/models.py#L237 .  The only way I know of to avoid getting prompted for a new migration whenever the file storage changes is to use the default file storage so it doesn't need to be specified in the field at all.  But last I heard, we explicitly don't want to use the default file storage for these task artifacts because we want to customize some of the S3 settings in stage and production for this particular use case (object expiration, etc.)